File: Remember_Java/src/main/java/work_with_arrays/ArrayTask.java
package work_with_arrays;

import java.util.Arrays;
import java.util.Scanner;

public class ArrayTask {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int[] arr = new int[n];
        for (int i = 0; i < n; i++) {
            arr[i] = scanner.nextInt();
        }
        printArray(sortAscending(arr));
        printArray(sortDescending(arr));
        System.out.println(maxElement(arr));
        System.out.println(minElement(arr));
        printArray(negativeNumbers(arr));
        printArray(evenNumbers(arr));
        printArray(numbersDivisibleByThree(arr));
        System.out.println(sumArrayElements(arr));
    }

    static void printArray(int[] arr) {
        for (int j : arr) {
            System.out.print(j + " ");
        }
        System.out.println();
    }

    public static int[] sortAscending(int[] arr) {
        int[] sortedArr = arr.clone();
        Arrays.sort(sortedArr);
        return sortedArr;
    }

    public static int[] sortDescending(int[] arr) {
        int[] sortedArr = arr.clone();
        Arrays.sort(sortedArr);
        for (int i = 0; i < sortedArr.length / 2; i++) {
            int temp = sortedArr[i];
            sortedArr[i] = sortedArr[sortedArr.length - 1 - i];
            sortedArr[sortedArr.length - 1 - i] = temp;
        }
        return sortedArr;
    }

    public static int maxElement(int[] arr) {
        return Arrays.stream(arr).max().getAsInt();
    }

    public static int minElement(int[] arr) {
        return Arrays.stream(arr).min().getAsInt();
    }

    public static int[] negativeNumbers(int[] arr) {
        return Arrays.stream(arr).filter(x -> x < 0).toArray();
    }

    public static int[] evenNumbers(int[] arr) {
        return Arrays.stream(arr).filter(x -> x % 2 == 0).toArray();
    }

    public static int[] numbersDivisibleByThree(int[] arr) {
        return Arrays.stream(arr).filter(x -> x % 3 == 0).toArray();
    }

    public static long sumArrayElements(int[] arr) {
        return Arrays.stream(arr).asLongStream().sum();
    }
}

File: src/main/java/Remember_Java/work_with_arrays/ArrayTask.cs
using System;
using System.Linq;

class ArrayTask
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        int[] arr = Array.ConvertAll(Console.ReadLine().Split(), int.Parse);
        PrintArray(SortAscending(arr));
        PrintArray(SortDescending(arr));
        Console.WriteLine(MaxElement(arr));
        Console.WriteLine(MinElement(arr));
        PrintArray(NegativeNumbers(arr));
        PrintArray(EvenNumbers(arr));
        PrintArray(NumbersDivisibleByThree(arr));
        Console.WriteLine(SumArrayElements(arr));
    }

    static void PrintArray(int[] arr)
    {
        Console.WriteLine(string.Join(" ", arr));
    }

    static int[] SortAscending(int[] arr)
    {
        int[] sortedArr = (int[])arr.Clone();
        Array.Sort(sortedArr);
        return sortedArr;
    }

    static int[] SortDescending(int[] arr)
    {
        int[] sortedArr = (int[])arr.Clone();
        Array.Sort(sortedArr);
        Array.Reverse(sortedArr);
        return sortedArr;
    }

    static int MaxElement(int[] arr)
    {
        return arr.Max();
    }

    static int MinElement(int[] arr)
    {
        return arr.Min();
    }

    static int[] NegativeNumbers(int[] arr)
    {
        return arr.Where(x => x < 0).ToArray();
    }

    static int[] EvenNumbers(int[] arr)
    {
        return arr.Where(x => x % 2 == 0).ToArray();
    }

    static int[] NumbersDivisibleByThree(int[] arr)
    {
        return arr.Where(x => x % 3 == 0).ToArray();
    }

    static long SumArrayElements(int[] arr)
    {
        return arr.Sum(x => (long)x);
    }
}
